src/components/pages/Profile/EditProfile.js

```diff
@@ -2,51 +2,39 @@ import React, { useState } from 'react';
 import { Form, Input, Button, Radio, Modal, TreeSelect } from 'antd';
 import '../../../styles/styles.css';
 import { connect } from 'react-redux';
+import useForms from '../../../hooks/useForms';
+
+const initialValues = {
```

Comment:
Is it necessary for these initialValues to have data within them? Or could they be an empty string?

Reply:
These temporary initial values need to be replaced by user profile data. I think we can pull it from the redux store but that would be for another ticket